src/libteam/0005-libteam-Add-warm_reboot-mode.patch

```diff
@@ -267,21 +282,21 @@ index 81324de..1d908db 100644
  static int lacp_callback_timeout(struct teamd_context *ctx, int events,
  				 void *priv)
  {
-@@ -1299,6 +1345,13 @@ static int lacp_port_added(struct teamd_context *ctx,
+@@ -1299,6 +1353,13 @@ static int lacp_port_added(struct teamd_context *ctx,
  	lacp_port_actor_init(lacp_port);
  	lacp_port_link_update(lacp_port);
  
 +	/* Read data from file and process it */
-+	if (ctx->warm_start && ctx->lacp_directory) {
++	if (ctx->warm_start && !lacp_port->has_read && ctx->lacp_directory) {
 +		(void)lacpdu_read(lacp_port);
-+		ctx->warm_start = false; 
```

Comment:
After removing this line, the warm_start flag will be always set to true after warm-start.
So all the newly-added ports will try to read the lacp pdu if they have not read yet.

Will it be problematic in some cases?
For example, if we delete the member port and add it back after warm-start. The port will try to read the file again?

Thanks,
Haiyang

Reply:
Hi Haiyang,Even if the port try to read the file again, there is no any file available to read. We remove the file right after we read it. Anyway I slightly changed logic in my last commit. Can you please check?

Reply:
Thanks Pavel for the clarification.  The new change LGTM.